Bottom: b7c9ab9afebd59ffe850562b4533d43102015e78
Top:    9004c781f11b8440c1c0e28eb17aa17f83e72ecc
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-12-04 15:56:13 +0000

Update netclass clearances

4.0 -> 4.5
0.2 -> 0.3


---

diff --git a/src/netclass.c b/src/netclass.c
index d13d3f0..54c6d6e 100644
--- a/src/netclass.c
+++ b/src/netclass.c
@@ -31,16 +31,16 @@ static char *netclass_names[10] =
 
 static Coord clearances[10][10] =
   {
-    {MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2), MM_TO_COORD (4.0)},
-    {MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (4.0), MM_TO_COORD (0.2)}
+    {MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3), MM_TO_COORD (4.5)},
+    {MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (4.5), MM_TO_COORD (0.3)}
   };
